aws/rds.md

```diff
@@ -0,0 +1,26 @@
+# AWS Relational Database Service (RDS)
+
+This doc presents guides on working with AWS RDS, Amazon's Database-as-a-Service provider.
+
+## Connecting with an EC2 instance
+
+In order to connect to an RDS instance from an EC2 instance, you'll need to configure the security rules for the database to allow access from the instance. Broadly speaking, AWS allows you to do this by 1) creating the database and the EC2 instance in the same Virtual Private Cloud (VPC) and 2) creating an inbound rule for the database's security group that permits access to PostgreSQL from the EC2 instance's security group.
```

Comment:
unrelatedly, are there examples of this practice in the wild, among the datamade repertoire? and/or, could you include brief guidance on when this practice is useful?

Reply:
Yup, added in a paragraph documenting the reasons for this approach and a link to an example project.